config/routes.rb

```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
+  mount_devise_token_auth_for 'User', at: 'api/v1/auth', skip: [:omniauth_callbacks]
   namespace :api do
     namespace :v0 do
-      resources :pings, only: [:index], constraints: { format: 'json' }
+      resources only: [:index], constraints: { format: 'json' }
```

Comment:
Not really sure why you removed this word. I would probably remove the whole name space now.